src/scripts/OSFramework/OSUI/Helper/Device.ts

```diff
@@ -140,9 +140,17 @@ namespace OSFramework.OSUI.Helper {
 		 * @memberof OSFramework.Helper.DeviceInfo
 		 */
 		private static _getUserAgent(userAgent = ''): string {
-			return userAgent.replace(' ', '') === ''
-				? window.navigator.userAgent.toLowerCase()
-				: userAgent.toLowerCase();
+			let localUserAgent = userAgent;
+
+			if (userAgent.replace(' ', '') === '') {
+				if (sessionStorage.previewDevicesUserAgent) {
+					localUserAgent = sessionStorage.previewDevicesUserAgent;
+				} else {
+					localUserAgent = window.navigator.userAgent;
+				}
```

Comment:
This has code specific for ODC. Will we keep it?

Reply:
@rugoncalves Same as above

Reply:
We must to, since it's piece of same code shared between platforms. Well, it's doable by using environment variables inside TypeScript, but we decided to not got that way for now since the code is not that much and do not have impact at O11.